README.md

```diff
@@ -1,8 +1,69 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Sai Prateek Reddy Annaiahgari . Jarvis Consulting
 
+I am a gradute of Electronics and Communication Engineering program with a specialization in Embedded Systems from Jawaharlal Nehru Technological Univarsity, Hyderabad, India. With a strong foundation in basic programming principles and analytical skills, I decided to explore the concepts of software engineering and expand my technical knowledge to pursue a career in Software Development. I have utilized various free online resources to learn different programming languages like python, C#, Java and Scala, and various tools by Apache Software Foundation. The improved knowledge helped me secure a job at Jarvis as a Data Engineer. At Jarvis, I learned to work as a team in a dynamic environment and develop efficient sofware solutions to complex real-world business problems. As a result of my abilty to quickly learn new technologies in a dynamic field like Data engineering, I will be a great fit to an equally dynamic team and organisation as a Data Engineer.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, python
+
+**Competent:** Scala, Apache Spark, Maven, Springboot, HTML/CSS, C#, REST API, Docker
+
+**Familiar:** Cloud Platforms - GCP and AWS, Kubernetes, Apache Kafka, Apache Hive, Apache Impala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_PrateekAnnaiahgari](https://github.com/jarviscanada/jarvis_data_eng_PrateekAnnaiahgari)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrateekAnnaiahgari/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring agent to log hardware specifications, and the resource usage of each node in the Linux cluster of 10 nodes, running on CentOS7. The agent uses two srcipts onr for host info and the other for host usage. The scripts send the data to PostGreSQL databse running on Docker. The usage data is collected every minute using Crontab. The collected data can be queried using SQL to answer specific resource usage questions.
```

Comment:
~of 10 nodes~

Reply:
~The agent uses two srcipts~

Developed a agent program using Linux Bash script to parse ....The shell script is able to insert data into a PostgreSQL database which is provisioned using docker..